app/serializers/activitypub/note_serializer.rb

```diff
@@ -41,7 +41,7 @@ def published
   end
 
   def url
-    TagManager.instance.url_for(object)
+    ActivityPub::TagManager.instance.uri_for(object)
```

Comment:
Is there a reason why you made this change? `ActivityPub…uri_for` should be used for the `id` and `TagManager…url_for` should be used for the `url` in an ideal situation, I think. If it was causing a test to fail then that likely means we need to change something else……